components/Palette/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './style.css';
+import { palettes } from '../../src/palettes';
+import { SchemeColor } from '../SchemeColor';
+
+export const PaletteData = ({ name, image, attribution, description }) => {
```

Comment:
Tahle komponenta je komponenta, která zobrazí jen jednu paletu. To máš dobře, posíláš si název palety, image, atd. Z indexu:
```
        {palettes.map((data) => (
          <PaletteData
            key={data.name}
            name={data.name}
            description={data.description}
            image={data.image}
            attribution={data.attribution}
            colors={data.colors}
          />
        ))}
```
akorát když si máš šáhnout pro barvy té jedné paletty, najednou to neděláš. Tj do toho volání v indexu ještě přidej
```
color={data.colors}
```
a pak ti to přijde tady
```
export const PaletteData = ({ name, image, attribution, description, colors }) => {
```
a už můžeš jen vypsat
```
        {colors.map((color) => (
          <SchemeColor key={color} color={color} />
        ))}
```
a komponenta SchemeColor jen pak zobrazí tu jednu barvu :)

Reply:
Protože teď zobrazuje jen první položku pole všech barev
```
export const SchemeColor = ({ colors }) => {
  return (
    <div className="scheme_colors">
      <div className="scheme_color" style={{ backgroundColor: colors[0] }}>
        {colors}
      </div>
    </div>
  );
};
```